taccsite_cms/settings.py

```diff
@@ -375,52 +357,6 @@ def get_subdirs_as_module_names(path):
     'easy_thumbnails.processors.filters'
 )
 
-
-# FEATURES.
-if CONSOLE_LOG_ENABLED:
-    print("--> Variable FEATURES: ")
-    for feature in FEATURES:
-        print(feature + ": ", FEATURES[feature])
-
-if current_secrets._FEATURES['blog']:
-    # Install required apps
-    INSTALLED_APPS += [
-        # Blog/News
-        # 'filer',              # Already added
-        # 'easy_thumbnails',    # Already added
-        'parler',
-        'taggit',
-        'taggit_autosuggest',
-        'meta',                 # also supports `djangocms_page_meta`
-        'sortedm2m',
-        'djangocms_blog',
-
-        # Metadata
-        'djangocms_page_meta',
-    ]
-
-    # Metadata: Configure
-    META_SITE_PROTOCOL = 'http'
-    META_USE_SITES = True
-    META_USE_OG_PROPERTIES = True
-    META_USE_TWITTER_PROPERTIES = True
-    META_USE_GOOGLEPLUS_PROPERTIES = True # django-meta 1.x+
-    # META_USE_SCHEMAORG_PROPERTIES=True  # django-meta 2.x+
-
-    # Blog/News: Set custom paths for templates
-    BLOG_PLUGIN_TEMPLATE_FOLDERS = (
-        ('plugins/default', 'Default template'),    # i.e. `templates/djangocms_blog/plugins/default/`
-        ('plugins/default-clone', 'Clone of default template'),  # i.e. `templates/djangocms_blog/plugins/default-clone/`
-    )
-
-    # Blog/News: Change default values for the auto-setup of one `BlogConfig`
-    # SEE: https://github.com/nephila/djangocms-blog/issues/629
-    BLOG_AUTO_SETUP = True
-    BLOG_AUTO_HOME_TITLE ='Home'
-    BLOG_AUTO_BLOG_TITLE = 'News'
-    BLOG_AUTO_APP_TITLE = 'News'
-
-
```

Comment:
Can these settings be added back in on a per portal basis using the `settings_custom` or `settings_local`, `settings_secret` files?

Reply:
This can remain in use, we just need to find a solution that implements a modular architecture. We can always work with CMD to ameliorate any potential issues

Reply:
I propose `settings_custom.example.py`, like Portal ([informal proposal](https://github.com/TACC/Core-CMS/pull/341#discussion_r708533776), see and open final bullet).

Reply:
- [Resolution to implement blog just for Texascale for now](https://github.com/TACC/Core-CMS/pull/341#discussion_r708623599)
- [Architectural reason to not have blog in Core](https://github.com/TACC/Core-CMS/pull/341#discussion_r708583857)